cpp/src/io/json/json_column.cu

```diff
@@ -625,6 +631,8 @@ void make_device_json_column(device_span<SymbolT const> input,
   auto ignore_vals = cudf::detail::make_host_vector<uint8_t>(num_columns, stream);
   std::vector<uint8_t> is_mixed_type_column(num_columns, 0);
   std::vector<uint8_t> is_pruned(num_columns, 0);
+  // for columns that are not mixed type but have been forced as string
+  std::vector<uint8_t> forced_as_string_column(num_columns, 0);
```

Comment:
Why not bool? To avoid the infamous std::vector<bool>?
AFAIK it should be fine as long as we don't try to get references to elements, which I think we don't do here.
If you really dislike std::vector<bool>, cudf::detail::host_vector<bool> is a good alternative.

Reply:
Done!